tensorflow-xla-runtime.spec

```diff
@@ -25,6 +25,8 @@ CXXFLAGS="-fPIC -Wl,-z,defs %{arch_build_flags} ${CMS_EIGEN_CXX_FLAGS} %{selecte
 pushd tensorflow/xla_aot_runtime_src
   # remove unnecessary implementations that use symbols that are not even existing
   find . -type f -path '*/service/cpu/runtime_fork_join.cc' | xargs rm -f
+  find . -type f -path '*/service/cpu/runtime_fft.cc' | xargs rm -f
+  find . -type f -path '*/service/cpu/runtime_single_threaded_fft.cc' | xargs rm -f
```

Comment:
@iarspider , the fft symbols are provided by tensorflow/thrid_part/ducc. Can you please add 
```
bazel $BAZEL_OPTS @ducc//:fft
bazel $BAZEL_OPTS @ducc//:fft_wrapper
```
in tensorflow-sources. This should generate 
```
> ls bazel-bin/external/ducc/*.so
bazel-bin/external/ducc/libfft.so  bazel-bin/external/ducc/libfft_wrapper.so
```
and we can then install these under tensorflow package and update xla_aot_runtime_src  to link against these libs